src/auxiliary/t8code.jl

```diff
@@ -54,37 +54,87 @@ function t8_free(ptr)
     T8code.Libt8.sc_free(t8_get_package_id(), ptr)
 end
 
-function trixi_t8_count_interfaces(forest)
-    # Check that forest is a committed, that is valid and usable, forest.
-    @assert t8_forest_is_committed(forest) != 0
+function t8_forest_ghost_get_remotes(forest)
+    num_remotes_ref = Ref{Cint}()
+    remotes_ptr = @ccall T8code.Libt8.libt8.t8_forest_ghost_get_remotes(forest::t8_forest_t,
```

Comment:
Why are these `@ccall`s necessary? Or phrased differently: Why are these functions not generated by T8code.jl?